react-native-purchases-ui/src/index.tsx

```diff
@@ -245,6 +249,7 @@ export default class RevenueCatUI {
     // safe area insets change. Not adding this extra padding on iOS will cause the content of the scrollview
     // to be hidden behind the rounded corners of the paywall.
     const [paddingBottom, setPaddingBottom] = useState(20);
+    const [height, setHeight] = useState(0);
```

Comment:
FWIW, In Flutter I had to give it an initial size to make it render in the screen or it wouldn't even show in the screen and try to measure. It might not happen in RN though.

Reply:
looks like it works fine in React Native 🤷‍♂️